lib/index.js

```diff
@@ -84,4 +84,24 @@ $("#getWeatherBtn").click(function(){
     $("body").css('background-image', `url(${backgroundImage})`)
   });
 });
+
+$("#addFavoriteBtn").click(function(){
+  event.preventDefault();
+  postFavorites(cityState);
+})
+
+var cityState = $("#locationInfo").val();
```

Comment:
I would move this variable inside your postFavorites function and then there would be no need to pass it as an argument.

Reply:
Great idea!  Just made this change!  Thanks, Dione!